brave/brave-core
# Leo annual subscription on Android

Resolves https://github.com/brave/brave-browser/issues/41008



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Make sure there is a trial period when purchasing Leo Yearly and monthly.
2. Make sure it is still premium after trial period ends.
